src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDCaseApi.java

```diff
@@ -113,13 +114,14 @@ public CCDCaseApi(
     }
 
     public List<Claim> getByDefendantEmail(String defendantEmail, String authorisation) {
+        LOGGER.debug("ccd search is by authorisation instead of defendant email {}", defendantEmail);
         User user = userService.getUser(authorisation);
-        return getAllCasesBy(user, ImmutableMap.of("case.defendantEmail", defendantEmail));
+        return getAllCasesBy(user, ImmutableMap.of());
     }
 
     public List<Claim> getByPaymentReference(String payReference, String authorisation) {
         User user = userService.getUser(authorisation);
-        return getAllCasesBy(user, ImmutableMap.of("case.claimData.payment.reference", payReference));
+        return getAllCasesBy(user, ImmutableMap.of("case.paymentReference", payReference));
```

Comment:
comment: just wondering why we dont call `ImmutableMap.of()` here too - as above?

Reply:
search criteria adds further filter after initial list of cases by  user authorisation

Reply:
This map contains the search criteria to filter the case.